### PR TITLE
Feat/1822 Create Squidex availability metric

### PR DESCRIPTION
**Pull request by niezgoda**
We want to create metric for when squidex responds with: 

- 5XX 
- 4XX
- GraphQL errors

## Requirements

- have a visual representation of squidex downtime (see list above) along with duration and frequency

### CODE REVIEW

File: apps/asap-server/src/autogenerated-gql/graphql.ts [hunk 1]
```diff
@@ -4231,17 +4231,13 @@ export type UsersDataSocialInputDto = {
 /** The structure of the Teams nested schema. */
 export type UsersDataTeamsChildDto = {
   id: Maybe<Array<Teams>>;
-  mainResearchInterests: Maybe<Scalars['String']>;
-  responsibilities: Maybe<Scalars['String']>;
   /** Attention: Check if this user needs to be added to Smart Simple */
```
Comment (kmaid) [inline apps/asap-server/src/autogenerated-gql/graphql.ts:4234]: Why is this in this PR

**Review by kmaid — CHANGES_REQUESTED** [review]

File: apps/asap-server/src/autogenerated-gql/graphql.ts (hunk 1 above)
Comment (niezgoda) [reply apps/asap-server/src/autogenerated-gql/graphql.ts:4234]: ask Piotr :D

**Review by peterstarling — CHANGES_REQUESTED** [review]
As I mentioned to your earlier - I think the dashboard definition should be part of the sls config and should be applied to all environments, it can be easily imported especially since our sls config is a ts file. 

You mentioned some limits though? If there is a limit it can be applied conditionally to dev and prod only

**Comment by peterstarling** [comment]
@niezgoda are we closing this?

**Pull request closed without merging**